arq/worker.py

```diff
@@ -275,11 +278,11 @@ def run(self) -> None:
 
             if self.burst:
                 if 0 <= self.max_burst_jobs <= self._jobs_started():
-                    await asyncio.gather(*self.tasks)
+                    await asyncio.gather(*list(self.tasks.values()))
                     return
                 queued_jobs = await self.pool.zcard(self.queue_name)
                 if queued_jobs == 0:
-                    await asyncio.gather(*self.tasks)
+                    await asyncio.gather(*(self.tasks.values()))
```

Comment:
```suggestion
                    await asyncio.gather(*self.tasks.values())
```
These parentheses do nothing.